### ISSUE TITLE
ChangeSignature: should trigger in positional records

### ISSUE THREAD

**Issue by jcouv** [dotnet/roslyn#44558]
ChangeSignature is currently not offered.
Note: we first need to clarify whether those are parameter or property declarations (tracked by https://github.com/dotnet/roslyn/issues/44556).

![image](https://user-images.githubusercontent.com/12466233/82926193-ae9e9b80-9f33-11ea-9c48-786bb50f4271.png)

Note that we offer ChangeSignature on other constructor parameters:
![image](https://user-images.githubusercontent.com/12466233/82926256-c83fe300-9f33-11ea-9896-faedfbf65c08.png)

Relates to https://github.com/dotnet/roslyn/issues/40726 (test plan for "records")

**Comment by jinujoseph** [comment]
cc @dpoeschl

**Comment by jcouv** [comment]
From discussion with @CyrusNajmabadi, let's take a stab for 16.8, otherwise push to 16.9.

**Comment by jcouv** [comment]
Assigned to @allisonchou since the duplicate issue had been assigned to her.
This may require new semantic model APIs to be able to update the base list (change signature on `Base` should update `record Derived(int i) : Base(...);`.
We should verify that callers are updated (`new Person(....)`), but we expect that to just work.

**Comment by allisonchou** [comment]
@jcouv For record declarations, is there a way to get their associated IParameterSymbols if they have any?

For example, with following code -
````C#
record Rec(string One, string Two)
{
}
````

`Rec` is an INamedTypeSymbol. However, I haven't been able to find a straightforward way to get parameter symbols from an INamedTypeSymbol, which we need for change signature.

**Comment by jcouv** [comment]
@allisonchou Tagging @CyrusNajmabadi who is working on symbol for associating parameters and properties in records.

**Comment by Youssef1313** [comment]
I'm working on this issue as part of a quick pass for IDE features that need to handle record doc comments.

**Comment by jcouv** [comment]
@allisonchou `GetDeclaredSymbol` on the parameter syntax will give you the corresponding parameter symbol.
